server/vulnerabilities/goval_dictionary/database.go

```diff
@@ -45,6 +45,8 @@ func (db Database) Eval(version fleet.OSVersion, software []fleet.Software) ([]f
 			var fixedVersionWithEpochPrefix, cve string
 			if err := affectedSoftwareRows.Scan(&fixedVersionWithEpochPrefix, &cve); err != nil {
 				return nil, fmt.Errorf("could not read package vulnerability result %s", swItem.Name)
+			} else if affectedSoftwareRows.Err() != nil {
```

Comment:
Given that I have the line above, why do I need this as well?

Reply:
these errs are separate from what's returned from Next().  Typically we check Err() after finishing the loop with Next()